config/packages/security.yaml

```diff
@@ -35,3 +35,6 @@ security:
             - ROLE_USER
             - TALK_SHOW
             - PARTICIPATION_SHOW
+            - SUBMIT_EDIT_ALL
+        ROLE_USER:
+            - SUBMIT_EDIT_OWN
```

Comment:
This is not really correct: instead of putting all the logic in the voter, you scatter it among the configuration and the application.

You should do the same thing as TALK_SHOW, PARTICIPATION_SHOW: used only `SUBMIT_EDIT`
And in the voter take some decision.

Finally, `SUBMIT_EDIT_ALL` seems not used.